test/freenet/support/JVMVersionTest.java

```diff
@@ -83,7 +83,7 @@ public void testCompare() {
 	        String v1 = orderedVersions[i];
 	        for (int j = 0; j < orderedVersions.length; j++) {
 	            String v2 = orderedVersions[j];
-	            int expected = Integer.signum(Integer.valueOf(i).compareTo(Integer.valueOf(j)));
+	            int expected = Integer.signum(Integer.valueOf(i).compareTo(j));
```

Comment:
`Integer` has a static `compare` method that you can use instead of still manually boxing one of the `int`s.